packages/bento-design-system/src/Button/Button.tsx

```diff
@@ -40,6 +42,7 @@ export function Button(props: Props) {
     // @ts-expect-error
     internal_unsafe__bypassUsePress,
   } = props;
+  const { formButtonProps } = useFormButtonProps(props);
```

Comment:
I see the logic behind this, but I wonder if it's not more practical to add something like `lodash.pick` (1kb gzipped) and turn this whole thing into:

```ts
const otherProps = pick(props, [
 	"name",
  "value",
  "form",
  "formAction",
  "formEncType",
  "formMethod",
  "formNoValidate",
  "formTarget",
	"onKeyDown",
	"onKeyUp"
]);
```

What do you think?

Reply:
yeah a little cleaner and keeps everything in Button.tsx,  i have pushed this chagne